client/src/modules/stock/inventory-adjustment/templates/difference.tmpl.html

```diff
@@ -0,0 +1 @@
+<div class="ui-grid-cell-contents">{{row.entity.old_quantity - row.entity.quantity}}</div>
```

Comment:
Not sure about this:  Shouldn't this be:  NEW - OLD?  That way positive values mean the inventory has increased and vice versa.

Reply:
Hey @buzunda, this is great!  I do agree with @jmcameron, it makes more sense to do `quantity - old_quantity`.  I believe this is what the user who requested the feature was looking for. 

Otherwise, perfect!

Reply:
ok, I will change this soon